packages/block-library/src/navigation-menu/edit.js

```diff
@@ -53,7 +53,7 @@ function NavigationMenu( {
 			<div className="wp-block-navigation-menu">
 				<InnerBlocks
 					allowedBlocks={ [ 'core/navigation-menu-item' ] }
-					renderAppender={ InnerBlocks.ButtonBlockAppender }
+					horizontalMover={ true }
```

Comment:
In my opinion it definitely makes sense to separate the position of the mover from the orientation:
```
<InnerBlocks
  __experimentalUseInlineMovers="before|after"
  __experimentalIsHorizontalLayout
/>
```
Something like that feels right to me. We already know for the gallery block that we'll need the option of having the mover before the block edit content if that is converted to inner blocks.

Reply:
yes @talldan that is exactly what I am currently implementing, a separation between the two. In thinking I arrived at the same conclusion, being that we will need all combos in various blocks simply because we style with CSS and DOM position matters.